docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -157,7 +157,7 @@ Migrating existing tables
 -------------------------
 
 The connector can read from or write to Hive tables that have been migrated to Iceberg.
-Currently, there is no Trino support to migrate Hive tables to Trino, so you will
+Currently, there is no Trino support for migrating Hive tables to Iceberg, so you will
```

Comment:
Since you are updating ...

```
There is no Trino support for migrating Hive tables to Iceberg, so you need either use the Iceberg API or Apache Spark.
```

Reply:
> There is no Trino support for migrating Hive tables to Iceberg, so you need either use the Iceberg API or Apache Spark.

I think we need one more word in here:

```
There is no Trino support for migrating Hive tables to Iceberg, so you need **to** either use the Iceberg API or Apache Spark.
```

Correct?

Reply:
yep